src/coreclr/utilcode/utilmessagebox.cpp

```diff
@@ -187,48 +187,46 @@ int UtilMessageBoxNonLocalizedVA(
         }
 
 #if !defined(FEATURE_UTILCODE_NO_DEPENDENCIES)
-        // If the current process isn't interactive (a service for example), then we report the message
-        // in the event log and via OutputDebugString.
+#ifdef HOST_UNIX
```

Comment:
```suggestion
#ifdef HOST_WINDOWS
```

Reply:
That isn't correct. The original was `#ifndef HOST_UNIX`, which make the macro `TRUE`. The `HOST_UNIX` branch was always `FALSE` and since the original `if` was negate it would only take the branch on `HOST_UNIX`.

Reply:
ah I missed the `!` in `!RunningInteractive` 😄